doc/api/os.md

```diff
@@ -243,9 +343,10 @@ added: v6.0.0
     If `encoding` is set to `'buffer'`, the `username`, `shell`, and `homedir`
     values will be `Buffer` instances. (Default: 'utf8')
 
-Returns a subset of the password file entry for the current effective user. The
-returned object includes the `username`, `uid`, `gid`, `shell`, and `homedir`.
-On Windows, the `uid` and `gid` fields are `-1`, and `shell` is `null`.
+The `os.userInfo()` method returns a subset of the password file entry for the
```

Comment:
minor nit: "password file (entry)" is a bit of UNIX-ism I think? Is there a more general term that would work better for Windows?